cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,78 @@
+import PageObject from '../PageObject';
+
+class SettingsPageObject extends PageObject {
+    url = '/#/settings';
+
+    get usernameField() {
+        return cy.getByDataCy('username-settings-field')
+    }
+
+    get bioField() {
+        return cy.getByDataCy('bio-settings-field')
+    }
+
+    get updateSettingsBtn() {
+        return cy.getByDataCy('update-settings-btn')
+    }
+
+    get emailField() {
+        return cy.getByDataCy('email-settings-field')
+    }
+
+    get passwordField() {
+        return cy.getByDataCy('password-settings-field')
+    }
+
+    get LogOutBtn() {
+        return cy.getByDataCy('logout-settings-btn')
+    }
+
+    typeUsername(updateUsername) {
+        this.usernameField
+            .type(updateUsername);
+    }
+
+    typeBio(updateBio) {
+        this.bioFieldField
+            .type(updateBio);
+    }
+
+    clickOnUpdateSettingsBtn() {
+        this.updateSettingsBtn
+            .click();
+    }
+
+    verifyUsernameUpdate(updateUsername) {
+        this.usernameField
+            .should('contain', updateUsername);
+    }
+
+    verifyBiolUpdate(updateBio) {
+        this.bioField
+            .should('contain', updateBio);
+    }
+
+    typeEmail(updateEmail){
+        this.emailField
+            .type(updateEmail);
+    }
+
+    verifyEmailUpdate(updateEmail) {
+        this.emailFieldField
```

Comment:
There is an incorrect reference in the `verifyEmailUpdate` method. It uses `this.emailFieldField` instead of `this.emailField`. Correct this to ensure the method functions properly.